### DISCUSSION TITLE
make: *** [Makefile:43: deploy] Error 1

### DISCUSSION THREAD

**Discussion by YavorJJ** [Cyfrin/foundry-full-course-cu#1827] [Q&A]
I can't set **make deploy** in the terminal because it gives me this error:

I am on lesson 2, FundMe Front End.

![Screenshot 2024-06-12 165232](https://github.com/Cyfrin/foundry-full-course-f23/assets/171678195/e50407ca-ca6d-4350-ab8b-8cae06b25d98)

if someone would take his time out to help me out here, much appreciated!🫡

**Comment by urosognjenovic** [comment]
Try adding ```.PHONY: deploy``` inside the makefile. If you already have ```.PHONY```, just add ```deploy``` to the list.

**Reply by YavorJJ** [reply]
just checked, I have it but it still does not work.

**Reply by urosognjenovic** [reply]
Could you please share your makefile with us?

**Reply by YavorJJ** [reply]
sure, here it is: 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/171678195/e0ab921f-ea29-43d3-b7e2-744a2d2d0096)

**Reply by EngrPips** [reply]
> sure, here it is: ![image](https://private-user-images.githubusercontent.com/171678195/339034663-e0ab921f-ea29-43d3-b7e2-744a2d2d0096.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyMDgzMTQsIm5iZiI6MTcxODIwODAxNCwicGF0aCI6Ii8xNzE2NzgxOTUvMzM5MDM0NjYzLWUwYWI5MjFmLWVhMjktNDNkMy1iN2UyLTc0NGEyZDJkMDA5Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMlQxNjAwMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04OTcxZDFiMDA3YmRjZGE3MWM3YmVkMTA0MzM2OWQyODBhYTIyMjBmNDBkMTFjMDFmYTU3ZmY0Nzk5YmI1MGU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.A-so_ZXMDe-cku92-ax7Ht8fpzHSXuGL-Iu64sfoSqk)

If This is the `Makefile` you showed in your working directory, then you need to make sure your terminal is pointing to the root folder of your directory and then run the make command again. It should work. Your root folder is `FOUNDRY-FUND-23`.

The image below shows me running my make command while the terminal points to my project root folder.

<img width="1440" alt="Screenshot 2024-06-12 at 17 07 01" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/c24c672b-fb0d-43ab-b4ee-b8f2c2f6ca6d">

**Reply by YavorJJ** [reply]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/171678195/f29ab9c5-2087-45c7-b7ff-cff3b184236a)

**Reply by YavorJJ** [reply]
I found my mistake, it was an error within the Makefile code. I am sorry if I wasted any of your time. Thank you, wish you the best.

**Reply by YavorJJ** [reply]
> Try adding `.PHONY: deploy` inside the makefile. If you already have `.PHONY`, just add `deploy` to the list.

I found my mistake, sorry for wasting your time :) 
Wish you the best :)

**Reply by EngrPips** [reply]
Excellent, what was the error I am curious @YavorJJ.

**Reply by YavorJJ** [reply]
> Excellent, what was the error I am curious @YavorJJ.

the error was I didn't set and write the Makefile code correctly like it was in Patrick's github. 
I went and saw that there was a difference between his and mine and changed it.

**Reply by EngrPips** [reply]
Oh, I see, Thanks for the response.

**Comment by EngrPips** [comment]
Hello @YavorJJ, Can you please show your working directory like the one I showed below?

<img width="347" alt="Screenshot 2024-06-12 at 15 28 34" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/802399a9-3f87-4568-8013-98b9e07689ba">

**Reply by YavorJJ** [reply]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/171678195/d95349af-313a-498b-8f11-a285a53239ee)

here it is!
